### PR TITLE
Minor UX Improvements 

### LINKED ISSUE #1609: Total policies number hard to read
### Describe the bug

The number of "Total policies" in the "Enabled Poliicies" section of the Dashboard are hard to read. The number is dark color that blends in the background.  

### Steps to reproduce

1. Go to Cloud Security dashboard

### Expected behavior

I would expect the number to be easily readable. 

### Environment

How are you deploying or using Panther?

- Panther version or commit: v1.9.0-RC1

### LINKED ISSUE #1617: Small UI/UX Bugs/Improvements from v1.9
### Bug 1

**Description:** The Alert Overview Rule Threshold attribute is set to time instead of int (1min vs 1)
**Reproducing**: Open any alert and look at the Rule Threshold
**Expected Behavior**: The Rule Threshold should just be an integer

### Bug 2

**Description:** The error box presented when attempting to save a rule with failing unit tests cannot be cleared
**Reproducing**: Open the rule editor, make a change to the rule which fails the unit tests, and try to save.
**Expected Behavior**: The error should have an 'x' to clear or should go away after a couple of seconds.

### Improvement 1

**Description:** On the log analysis page, there is no specification of what the "last period was" in the alert count box
**Reproducing**: View the total alert count box in the log analysis overview
**Expected Behavior**: We should display what the last period was (3d)

### Environment

Panther v1.9.0-RC1

### PR DESCRIPTION
**Pull request by alexmylonas**
## Background

This PR fixes some small UX issues, we collected over the last days.

Closes #1617
Closes  #1609 

## Changes

- [x] Errors on create/edit Rule are now discardable
- [x] Alerts displayed on Log Analysis overview are only `Open` || `Triaged`
- [x] Blocking deselection of the last series from the legend for Timeline charts

## Testing

- Locally

### CODE REVIEW

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 1]
```diff
@@ -323,14 +323,18 @@ const TimeSeriesChart: React.FC<TimeSeriesLinesProps> = ({
...
+          console.log('if', currentSelected);
           const newSelection = {};
           Object.keys(selected).forEach(key => {
             newSelection[key] = key === name;
           });
           chartOptions.legend.selected = newSelection;
-        } else {
+
+          // This checks if everything is deselected, if not we can go ahead and update selection
+        } else if (!Object.keys(selected).every(key => !selected[key])) {
```
Comment (3nvi) [inline web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:337]: I would expect the following behaviour:

When 1 item is selected, if it's clicked again, it resets back to the original state (selecting all items)

**Review by 3nvi — COMMENTED** [review]
Minor changes and g2g

File: web/src/pages/EditRule/EditRule.tsx [hunk 2]
```diff
@@ -118,6 +118,7 @@ const EditRulePage: React.FC = () => {
         <Box mt={2} mb={6}>
           <Alert
             variant="error"
+            discardable
```
[the remark this answers is not in the document]
Comment (alexmylonas) [reply web/src/pages/EditRule/EditRule.tsx:121]: Done!

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 3]
```diff
@@ -323,14 +323,18 @@ const TimeSeriesChart: React.FC<TimeSeriesLinesProps> = ({
       newChart.on('legendselectchanged', function (obj) {
         const { selected, name } = obj;
         const currentSelected = chartOptions.legend.selected;
+        console.log('log', selected, currentSelected);
```
[the remark this answers is not in the document]
Comment (alexmylonas) [reply web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx]: Good callout, added the rule for eslint.

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx (hunk 1 above)
Comment (alexmylonas) [reply web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx]: Updated!

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 4]
```diff
@@ -330,6 +330,13 @@ const TimeSeriesChart: React.FC<TimeSeriesLinesProps> = ({
             newSelection[key] = key === name;
           });
           chartOptions.legend.selected = newSelection;
+
+          // This checks if everything is going to deselected, if yes we enable all series
+        } else if (Object.keys(selected).every(key => !selected[key])) {
```
Comment (3nvi) [inline web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:335]:
```suggestion
        } else if (!Object.keys(selected).some(key => selected[key])) {
```

optional mod for readability

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 5]
```diff
@@ -330,6 +330,13 @@ const TimeSeriesChart: React.FC<TimeSeriesLinesProps> = ({
...
+          chartOptions.legend.selected = Object.keys(selected).reduce((acc, key) => {
```
Comment (3nvi) [inline web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:336]: can we follow the same convention in both `if` / `else`?

Meaning that the same style of code should be present here and in the `if` above:

```tsx
const newSelection = {};
          Object.keys(selected).forEach(key => {
            newSelection[key] = key === name;
          });
          chartOptions.legend.selected = newSelection;
```

**Review by 3nvi — APPROVED** [review]
LGTM 🎉 

Approving, but ideally I'd like to see similar coding style within the same  `if`

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx (hunk 4 above)
Comment (alexmylonas) [reply web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx]: Updated!

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx (hunk 5 above)
Comment (alexmylonas) [reply web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:334]: Updated!

**Pull request merged**